slackEvents.js

```diff
@@ -1,18 +1,23 @@
+'use strict';
+
 const slackEventsApi = require('@slack/events-api');
 const { WebClient } = require('@slack/client');
 
 const slackEvents = slackEventsApi.createSlackEventAdapter(process.env.SLACK_VERIFICATION_TOKEN, { includeBody: true });
 
-slackEvents.on('message', (message, body) => {
-  if (!message.subtype && message.text === 'hi') {
+slackEvents.on('app_mention', (message, body) => {
+  // Only respond to messages that have no subtype (plain messages)
+  if (!message.subtype) {
     const slack = new WebClient(process.env.SAPPHIRE_OAUTH_TOKEN);
     
     if (slack) {
-      slack.chat.postMessage({ 
-        channel: message.channel, 
-        text: `B-b-baka <@${message.user}>! :blushkek:` 
-      })
-        .catch(console.error);
+      if (message.text.indexOf('hi') >= 0) {
```

Comment:
Makes more sense to me to first check that we have a message to post before we create an instance of WebClient.

Reply:
Yea and we should probably build a SlackClient class in the future

Reply:
This is preparing for future tickets - the events have already been filtered down to the ones where Sakura has been mentioned, so I think there should be a need to respond to each.

Otherwise it wouldn't make sense to repeat a bunch of WebClient instantiations between each future condition block.